src/ngraph_builder.cc

```diff
@@ -81,6 +81,16 @@ static void SaveNgOp(Builder::OpMap& ng_op_map, const std::string& op_name,
   // no need to try-catch, map[key] will create vector object
   // if not exists
   ng_op_map[op_name].push_back(output_node);
+  // TODO (sarkars): remove try catch when ngraph is updated
+  try {
+    output_node->set_friendly_name(op_name + "_" +
+                                   to_string(ng_op_map[op_name].size()));
+  } catch (const ngraph::ngraph_error& e) {
```

Comment:
Do we want to do size()-1 ? To match the indexing output indexing in TF?

Reply:
yes, of course!

Reply:
May be use "[]" for indexes. Also add a unit test to test the name setting